lib/guardrails/openai/multiple_checker.rb

```diff
@@ -0,0 +1,50 @@
+module Guardrails
+  module OpenAI
+    class MultipleChecker
+      OPENAI_MODEL = "gpt-4o-mini".freeze
+
+      def self.call(...) = new(...).call
+
+      def initialize(input, prompt, max_tokens)
+        @input = input
+        @prompt = prompt
+        @max_tokens = max_tokens
+        @openai_client = OpenAIClient.build
+      end
+
+      def call
+        llm_response = openai_response.dig("choices", 0)
+        llm_guardrail_result = llm_response.dig("message", "content")
+        llm_token_usage = openai_response["usage"]
+
+        {
+          llm_response:,
+          llm_guardrail_result:,
+          llm_token_usage:,
+        }
```

Comment:
Very minor but you can save a few lines here:

```suggestion
        {
          llm_response: openai_response.dig("choices", 0)
          llm_guardrail_result: llm_response.dig("message", "content")
          llm_token_usage: openai_response["usage"],
        }
```